src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,5 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="com.amplitude">
```

Comment:
will this affect the distribution of our SDK? when we release to Maven, and when customers pull the SDK, will this affect their experience in any way?

Reply:
No, it won't. It is specific to us.